src/chrome/content/rules/strasbourgfestival.com.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="strasbourgfestival.com">
+
+        <target host="strasbourgfestival.com" />
+
+        <rule from="^http:"
+                to="https:" />
+
+	<test url="http://strasbourgfestival.com/" />
+
+</ruleset>
```

Comment:
```suggestion
<ruleset name="strasbourgfestival.com">
	<target host="strasbourgfestival.com" />
	<target host="www.strasbourgfestival.com" />

	<rule from="^http:" to="https:" />
</ruleset>
```

Reply:
Hello,
I am beginner with Git and I do not know how to modify an already-made commit or push.